core/src/main/java/org/apache/iceberg/util/PropertyUtil.java

```diff
@@ -74,6 +76,19 @@ public static String propertyAsString(Map<String, String> properties,
     return defaultValue;
   }
 
+  public static List<Long> propertyAsLongList(Map<String, String> properties,
+                                              String property) {
+    String value = properties.get(property);
+    if (value == null) {
+      return null;
+    }
+    return Splitter.on(",")
```

Comment:
Nit: Might be good to declare the splitter one time as a private static field, as opposed to constructing it every time (unless there's some serialization-related reason it has to be reinstantiated).

Reply:
+1 to Kyle's suggestion